es-vue-base/src/lib-utils/nav-bar-global-content.js

```diff
@@ -387,7 +387,7 @@ export default (
                 },
                 {
                     name: 'Get personalized quotes',
-                    link: `${HEAT_PUMPS_DOMAIN}/dashboard`,
+                    link: `${ES_DOMAIN}/shop/heat-pumps/`,
```

Comment:
This link is meant to take the user straight into onboarding, just like the similar bold link in the nav under Home Solar takes you to https://www.energysage.com/market/start/ - would it make sense to have a similar link destination here, with the appropriate heat pumps parameter like https://www.energysage.com/onboarding/start/?product=heatpump, assuming that, like 1QPS for solar onboarding, zip code can be captured within the flow itself if not initially provided?

Reply:
Good question Eric! 
If we direct users to straight into onboarding from this link here, we would have to add a zip code capture in some way to enable NexStar. 
This has been discussed in the comments in[ this ticket](https://energysage.atlassian.net/browse/IDR-425), and the solution to point to the [HP homepage](https://int.www.energysage.dev/shop/heat-pumps/) has been chosen, as it will accommodate the zip code and will route users to the NexStar flow if they qualified.

Reply:
Ok fair! Just wanted to check.

Reply:
Thank you for checking!